koku/api/report/view.py

```diff
@@ -129,6 +131,11 @@ def get_report(self, provider, report):
         for rep in reports:
             if report == rep.get('report'):
                 return rep
+
+        if report == 'default':
```

Comment:
Trying to understand this part.  Where is `_generic_report` passing in `report='default'` to hit this case?

Reply:
It's not. 

`_generic_report` passes in some string, say `foo`, and if the string falls through the for-loop, this function will recurse to look for the default. But we only want it to recurse once, not an infinite number of times. :slightly_smiling_face: